### PR TITLE
Fix polish search (bug 1133948)

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

**Comment by diox** [comment]
In `.travis.yml` we are using `1.3.7` ; what version are we using in prod/dev/stage ?

**Comment by robhudson** [comment]
I believe we're on 1.3.7 but 1.3.8 is a security update to 1.3.7. I'll update the travis.yml as part of the doc fix.

File: mkt/search/tests/test_filters.py [hunk 1]
```diff
@@ -65,6 +67,24 @@ def test_no_fuzzy_multi_word(self):
...
         ok_('fuzzy' not in qs_str)
 
+    @override_settings(ES_USE_PLUGINS=True)
+    def test_polish_analyzer(self):
+        """
+        Test that the polish analyzer is included correctly since it is an
+        exception to the rest b/c it is a plugin.
+        """
+        with self.activate(locale='pl'):
+            qs = self._filter(data={'q': u'próba', 'lang': 'pl'})
```
Comment (robhudson) [inline mkt/search/tests/test_filters.py]: oh, `lang` here isn't needed b/c of the locale activation that was needed above. I'll remove.

**Comment by robhudson** [comment]
Updated. New and improved! Now with travis and more tests!

**Comment by diox** [comment]
I haven't tested this locally but it looks good to me. r+

**Comment by robhudson** [comment]
Merged

**Pull request closed without merging**